src/java.base/share/classes/java/nio/file/FileSystem.java

```diff
@@ -304,7 +304,7 @@ protected FileSystem() {
      * <blockquote><pre>
      * <i>syntax</i><b>:</b><i>pattern</i>
      * </pre></blockquote>
-     * where {@code ':'} stands for itself.
+     * where <i>syntax</i> must not be empty and {@code ':'} stands for itself.
```

Comment:
We could flesh this out a bit more with something like:

"where <i>syntax</i> is the non-empty name of the syntax,  <i>pattern</i> is a possibly-empty pattern string, and {@code ':'} stands for itself."

Reply:
That is better.

Reply:
So changed in b9de2604b74c1082c6b0fa10435405120b971e93.